templates/news/2016/12/19/ubuntu-core-iot.html

```diff
@@ -0,0 +1,91 @@
+{% extends "news/base_news.html" %}
+{% block title %}和Ubuntu Core的一场黑马 - 将物联网的进化革命进行到底{% endblock %}
+{% block content %}
+
+<div class="row row-hero no-border">
+    <div class="eight-col">
+        <h1>和Ubuntu Core的一场黑马 - 将物联网的进化革命进行到底</h1>
+        <div class="twelve-col">
+            <img src="{{ STATIC_URL }}img/news/jam-banner.jpg" alt="Ubuntu Jam banner" />
+        </div>
+        <p>为了庆祝Ubuntu Core 16正式发布，我们在IoT领域高手云集的深圳，专为欲大展拳脚的各路开发高手，创客团队举办了主题为“Jamming with Ubuntu Core”的黑客松活动。本次活动也是第一次以Ubuntu Core为开发主题的开发活动。活动吸引了来自不同行业的软硬件开发者踊跃参与，他们也通过Ubuntu Core展示了他们独特且创意的作品。</p>
+        <div class="twelve-col">
+            <img src="{{ STATIC_URL }}img/news/jam-1-skyline.jpg" alt="Shenzhen skyline" />
+        </div>
+        <p>目前，Ubuntu Core 在柜顶交换机、<a href="https://www.ubuntu.com/internet-of-things/gateways?utm_campaign=Device_FY17_IOT_China_ShenzhenUC16launchHackathon&utm_medium=eventpost&utm_source=huodongxing" class="external">工业网关</a>、家庭网关、无线接入网络、<a href="https://www.ubuntu.com/internet-of-things/digital-signage?utm_campaign=Device_FY17_IOT_China_ShenzhenUC16launchHackathon&utm_medium=eventpost&utm_source=huodongxing" class="external">数字标牌</a>、<a href="https://www.ubuntu.com/internet-of-things/robotics?utm_campaign=Device_FY17_IOT_China_ShenzhenUC16launchHackathon&utm_medium=eventpost&utm_source=huodongxing" class="external">机器人和无人机</a>等领域得到了广泛应用。部分活动开发者也来自于上述行业，在36小时的hack时间内，创意和作品将是他们对物联网的完美诠释。</p>
+        <div class="twelve-col">
+            <img src="{{ STATIC_URL }}img/news/jam-2-illustration.png" alt="Ubuntu Core illustration" />
+        </div>
+        <p>与以往Hackathon活动相比，这次Ubuntu Hackathon多了一些不同，因为不同从而产生不同创意之作。本次活动迎来了远从台北而来的QNAP开发者，致力于在Ubuntu Core上打造IoT与NAS存储器的应用解决方案，同时也有上一届Hackathon的参赛者，更重要的是赢了很多新的开发者参与。他们的精彩项目也为本次黑客松活动添加了特别的气氛。</p>
+
+        <h2>作品及项目：短短36小时，开发者们为Ubuntu Core带来了8个佳作。他们分别是：</h2>
+        <h3>1、天台聚落 （Jarvis Chung，Lucas Lu  ）</h3>
+        <h4>项目介绍：（snap：water-iot-service）</h4>
+        <p>天台聚落是指在IoT 物联⽹环境下监控⽔质，在不同环境下实现远程监控对水质的监控，解决了因为环境问题而不能时刻掌握水质的困境。该项目利用Ubuntu Core稳定且时刻保持最新的优势将传感器收集的信息传输到NAS再通过web服务显示在前端，实现了数据的采集和分析。</p>
+        <p>此项目的愿景：为水资源的监测和管理提供更加便捷、直观、科学的思路，可用水库、蓄水池、水塔等无人照看且检修、监测困难水资源管理领域。</p>
+        <h4>团队介绍：</h4>
+        <p><a href="https://www.qnap.com/zh-cn" class="external">威联通科技股份有限公司</a> (QNAP Systems, Inc.) ，公司专注于提供专业级的网络附加存储（NAS）和网络视频录像机（NVR）解决方案, 威联通的集成技术和设计带来优质的产品，其文件共享，虚拟化应用程序，存储管理和商业环境检测有效地提高企业运行效率。</p>
+        <div class="twelve-col">
+            <img src="{{ STATIC_URL }}img/news/jam-4.jpg" alt="Hackathon photo" />
```

Comment:
This image doesn't work as the file has an extension of `JPG`. Probably worth lowercasing the extension on the file, rather than changing it here.